be/src/Project/Services/Ordering/Ordering.Api/Services/OrdersService.cs

```diff
@@ -864,20 +864,23 @@ private async Task MapStatusesToOrderStatusId(Guid? orderId)
                         lastOrderItemStatusChanges.Add(new OrderItemStatusServiceModel
                         {
                             OrderItemStateId = lastOrderItemStatusChange.OrderItemStateId,
-                            OrderItemStatusId = lastOrderItemStatusChange.OrderItemStatusId
+                            OrderItemStatusId = lastOrderItemStatusChange.OrderItemStatusId,
+                            CreatedDate = lastOrderItemStatusChange.CreatedDate
```

Comment:
**question:** wouldn't CreatedDate = DateTime.UtcNow be enough?

Reply:
it was for testing, i removed